src/language/variables.md

```diff
@@ -58,7 +58,7 @@ assert(lineCount == null);
   _condition_ is false. For details, check out [Assert][].
 {{site.alert.end}}
 
-If you enable null safety, then you must initialize the values
+With null safety, you must initialize the values
```

Comment:
I think we need to tweak this page a bit more. This is, I think, the perfect early point to introduce null safety at is most basic level. I think we need to pull a bit of intro content into this section about variables.